R/pkg/tests/fulltests/test_sparkSQL.R

```diff
@@ -1382,8 +1382,8 @@ test_that("column functions", {
   c20 <- to_timestamp(c) + to_timestamp(c, "yyyy") + to_date(c, "yyyy")
   c21 <- posexplode_outer(c) + explode_outer(c)
   c22 <- not(c)
-  c23 <- trunc(c, "year") + trunc(c, "yyyy") + trunc(c, "yy") +
-    trunc(c, "month") + trunc(c, "mon") + trunc(c, "mm")
+  c23 <- trunc(to_date(c), "year") + trunc(to_date(c), "yyyy") + trunc(to_date(c), "yy") +
+    trunc(to_date(c), "month") + trunc(to_date(c), "mon") + trunc(to_date(c), "mm")
```

Comment:
that's a good point. fortunately (?) trunc was only added to R in 2.3.0, so I think we need to make sure (manually, add unit test) that trunc works on date columns and numeric columns